src/Core/Models/Table/OrganizationUser.cs

```diff
@@ -6,18 +6,43 @@ namespace Bit.Core.Models.Table
 {
     public class OrganizationUser : ITableObject<Guid>, IExternal
     {
+        [DbOrder(1)]
```

Comment:
I'm not quite sure I understand where Dapper is failing on this mapping structure to match a custom UDT in SQL... this is getting a little verbose/gnarly for me. Let's discuss tomorrow perhaps and see if there are any other ways to work through this.

Reply:
I agree. I looked for a better solution, but I couldn't find something that intelligently aligns tables to objects. I didn't specifically look at Dapper, though. I hope there's something better

Reply:
It appears that Dapper requires the data table it's producing a TVP from to have the same column order as the data object.

The order that properties are pulled from a class by reflection (and thus the column order made by my helper) is either non-deterministic or in code-order, I'm not sure which, but neither suffices.

We could just continue making individual helpers on a TVP-type basis. I was just trying to implement something that is more generically useful.